src/Model/InscriptionManager.php

```diff
@@ -26,13 +26,15 @@ public function __construct()
 
     public function selectValidate()
     {
-        $query = ('SELECT * FROM ' . $this->table . ' WHERE is_validate = true');
+        $query = ('SELECT * FROM ' . $this->table . ' WHERE is_validate = true ' .
+            ' ORDER BY status is NULL, status, lastname');
```

Comment:
pourquoi `ORDER BY status is NULL, status, lastname` ? 
ça serait pas plutot `ORDER BY status, lastname` ?

Reply:
en fait ça me triait le status par ordre alphabétique, même les NULL, et en faisant ça j'avais bien les NULL à la fin et au début les membres avec un statut